### DISCUSSION TITLE
How to use Android's jar? And there are armeabi v7a dependencies on so files.

### DISCUSSION THREAD

**Discussion by bbhxwl** [dotnet/maui#22139] [Q&A]
How to use Android's jar? And there are armeabi v7a dependencies on so files.

![image](https://github.com/dotnet/maui/assets/26525599/10bab229-a7d7-4a4d-be09-0bf75f59451c)

**Comment by xtuzy** [comment]
create a [java binding library](https://learn.microsoft.com/en-us/xamarin/android/platform/binding-java-library/binding-a-jar), and create a [lib folder ](https://github.com/xtuzy/Veldrid.SPIRV.NativeAsset/tree/master/Veldrid.SPIRV.AndroidNativeAsset)to store .so
